README.md

```diff
@@ -71,6 +71,7 @@ Our `export` method provides support for the following options:
             - Can range from 0 to 255
         - `[3]` - Alpha/transparency value for background
             - Can range from 0 to 255
+	- quality `Number` - quality percentage of jpeg output from 0 to 100
```

Comment:
It looks like we indented with tabs instead of spaces. Can we use spaces for consistency with the rest of this document?